tests/system/test_instance_admin.py

```diff
@@ -87,6 +91,7 @@ def _modify_app_profile_helper(
     assert alt_profile.description == description
     assert alt_profile.routing_policy_type == routing_policy_type
     assert alt_profile.cluster_id == cluster_id
+    assert alt_profile.multi_cluster_ids == multi_cluster_ids
```

Comment:
can we add a few more system tests that create an app profile with multi cluster routing and ids specified, and also an update test adding or removing a cluster id?

Reply:
I've added the test below: `test_instance_create_app_profile_create_with_multi_cluster_ids`